apps/src/sites/studio/pages/components/LevelTokenDetails.jsx

```diff
@@ -0,0 +1,153 @@
+/* global levelKeyList */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
```

Comment:
Is this the first place we're import css? I'm a little surprised our webpack even knows how to do this.